index.html

```diff
@@ -57,8 +57,6 @@ <h2>Languages</h2>
                 
                 <li class="toctree-l1"><a class="reference internal" href="html.html">HTML</a></li>
                 
-                <li class="toctree-l1"><a class="reference internal" href="json.html">JSON</a></li>
```

Comment:
Not sure why this is removed, I did not see JSON on the master branch either. But it is on the demo site.

Reply:
My bad, I forgot to push to master. I have done that now, can you please rebase? Also, please commit your changes to master. I will rebuild the website after merging. :)